Lib/test/test_doctest/test_doctest2.py

```diff
@@ -108,6 +110,15 @@ def clsm(cls, val):
 
 
 class Test(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.colorize = _colorize.COLORIZE
+        _colorize.COLORIZE = False
+
+    def tearDown(self):
+        super().tearDown()
+        _colorize.COLORIZE = self.colorize
+
```

Comment:
It looks like tests pass for me locally if I delete this -- what's it for, exactly?

Reply:
Hmm, same here... Reverted!